doc/tools/coqrst/notations/parsing.py

```diff
@@ -14,12 +14,14 @@
 from antlr4.error.ErrorListener import ErrorListener
 from antlr4.Recognizer import Recognizer
 
+import os
+
 # Only print ANTLR version warning once
 checked_version = False
 check_version = Recognizer.checkVersion
 def checkVersion_once(*args, **kwargs):
     global checked_version
-    if not checked_version:
+    if not checked_version and os.getenv ("COQ_DEBUG_REFMAN"):
```

Comment:
Is this an unrelated change meant to remove the annoying ANTLR warning that we get at each build? In this case, wouldn't it make more sense to make `checked_version = True` if `COQ_DEBUG_REFMAN` is not defined, instead of repeatedly checking the value of this env var?